website/content/docs/common-workflows/workflow-ssh-proxycommand.mdx

```diff
@@ -0,0 +1,111 @@
+---
+layout: docs
+page_title: SSH ProxyCommand Workflow 
+sidebar_title: SSH ProxyCommand Workflow
+description: How to manage SSH proxy configuration with Boundary
+---
+
+# SSH ProxyCommand Workflow 
+
+The most common pattern for using Boundary to SSH is with the built-in `boundary connect ssh` command. However,
+there are more idomatic approaches that can be employed to make Boundary transparent to users, and at the same
+time simplify common developer and operator workflows. Using `ProxyCommand` to execute a proxy when invoking the
+SSH client is a common practice. In this workflow, we'll cover configuring your SSH client to execute the 
+`boundary` command, enabling a simplified SSH workflow that leverages Boundary's authenticated proxy for accessing
+targets.
+
+## SSH Configuration
+
+Start by configuring a `Host` entry in `.ssh/ssh_config` for `localhost`:
```

Comment:
To me this doesn't feel like the right pattern. Rather than have to have people do the manual "authorize the session" stuff below, why not create an SSH entry for each service where you specify the Hostname, and have the target info baked in? E.g.
```
Host boundary-web
    Hostname localhost
    ProxyCommand sh -c "boundary connect -target-id ttcp_XXXXXXXXXX -exec nc -- {{boundary.ip}} {{boundary.port}}"
```

Reply:
We can do that to, but you'll still need to authenticate and I didn't initially like the idea of hard coding the target ID. That being said, the target ID rarely if ever changes, so maybe it's not that bad - how about we show both?

Reply:
Another possibility is:

```
Host ttcp_*
  ProxyCommand sh -c "boundary connect -target-id %h -exec nc -- {{boundary.ip}} {{boundary.port}}"
```
This straddles the line between the comment above and the things we discussed -- it takes a host that almost certainly is a boundary target and uses the ProxyCommand without it taking over non-Boundary targets.

Reply:
> Thanks for the feedback @jorhett - the `localhost` example is meant to be an example. I assume most if not all users will want that to be set differently in a real-world scenario.

As I'm sure you well know from the support queries, the vast majority of people don't understand the layers involved. This leads me to defensively try and make all examples self-describing in a way that I can see what they are doing when they give me their example, whereas with everything under `localhost` you're gonna see a lot of things that look the same, and many of your users won't grasp those differences.

...and I also find target-oriented examples tend to be easier for people to grasp than ones that focus on the implementation rather than the goal     (speaking as a tech author)

Reply:
> it takes a host that almost certainly is a boundary target

given underscores aren't valid in a domain name, it's either that or an SSH config entry, sure. I think this is a totally useful and valid config if implemented in a wrapper that looks up the target id. But in testing with users, it would appear that directly using target IDs isn't going to be accepted at all.  And they're competent engineers.